.github/workflows/build.yml

```
@@ -43,7 +43,7 @@ jobs:
             certificate-secret: WINDOWS_SIGNING_CERTIFICATE_PFX # Name of the secret that contains the certificate.
             certificate-password-secret: WINDOWS_SIGNING_CERTIFICATE_PASSWORD # Name of the secret that contains the certificate password.
             certificate-extension: pfx  # File extension for the certificate.
-          - os: ubuntu-18.04 # https://github.com/arduino/arduino-ide/issues/259
+          - os: ubuntu-latest
```

```suggestion
          - os: ubuntu-20.04
```

This would be less impactful. This runner should remain available for some years to come.